languages/rust/oso/src/lib.rs

```diff
@@ -122,3 +122,19 @@ extern crate oso_derive;
 #[cfg(feature = "derive")]
 #[doc(hidden)]
 pub use oso_derive::*;
+
+#[cfg(feature = "uuid")]
+use uuid::Uuid;
+#[cfg(feature = "uuid")]
+impl PolarClass for Uuid {
+    fn get_polar_class_builder() -> ClassBuilder<Uuid> {
+        Class::builder()
+            .name("Uuid")
+            .set_equality_check(|a, b| a == b)
```

Comment:
Minor:
```suggestion
            .with_equality_check()
```

Can do this as a shortcut for types that impl `PartialEq`: https://docs.rs/oso/0.8.2/oso/struct.ClassBuilder.html#method.with_equality_check